rocky/reports/views/generate_report.py

```diff
@@ -100,6 +103,21 @@ class ExportSetupGenerateReportView(GenerateReportStepsMixin, BreadcrumbsGenerat
     breadcrumbs_step = 6
     current_step = 4
 
+    def post(self, request, *args, **kwargs):
+        selected_plugins = request.POST.getlist("plugin", [])
+
+        client = get_katalogus(self.organization.code)
+        for selected_plugin in selected_plugins:
+            try:
+                client.enable_boefje_by_id(selected_plugin)
```

Comment:
We should also check if the user has the permissions to enable these boefjes.

Reply:
to check permissions use:
https://github.com/minvws/nl-kat-coordination/blob/193764c28bf1904be2c7ca4d55e0884f7936b92e/rocky/account/mixins.py#L167

Reply:
This permission checks if the user can raise the clearance level of an object right? We want to enable plugins. Shouldnt this be another permission?

Reply:
Afaik enabling boefjes *is* another permission?

Reply:
I used `tools.can_enable_disable_boefje`, it seemed the most fitting :D

Reply:
excellent